src/Kotlin/KotlinLanguageService.kt

```diff
@@ -171,24 +164,46 @@ class KotlinLanguageService : LanguageService {
         }
     }
 
+    private fun ContentNode.renderAnnotationsForNode(node: DocumentationNode) {
+        node.annotations.forEach {
+            renderAnnotation(it)
+        }
+    }
+
+    private fun ContentNode.renderAnnotation(node: DocumentationNode) {
```

Comment:
I've looked at the generated docs now, and I think that they're easier to read without square brackets. I also plan to make it possible to exclude certain annotations (e.g. inlineOptions) from the output.